src/middleware/devtools.ts

```diff
@@ -170,16 +170,14 @@ export function devtools<
     }
     const devtoolsOptions =
       options === undefined
-        ? { name: undefined, anonymousActionType: undefined }
+        ? {}
         : typeof options === 'string'
         ? { name: options }
         : options
 
     let extensionConnector
     try {
-      extensionConnector =
-        (window as any).__REDUX_DEVTOOLS_EXTENSION__ ||
-        (window as any).top.__REDUX_DEVTOOLS_EXTENSION__
+      extensionConnector = window.__REDUX_DEVTOOLS_EXTENSION__
```

Comment:
Not sure why we need `top`. The official '@redux-devtools/extension' doesn't seem to have one.
This is breaking too.

Reply:
Probably so that the devtools work in codesandbox etc (as `__REDUX_DEVTOOLS_EXTENSION__` would only be in the parent window)

Reply:
hmm, let me experiment. (jotai/valtio doesn't do this, so we want to make three consistent.)

Reply:
https://codesandbox.io/s/cool-sun-bnowv5
`window.top` doesn't work anyway in codesandbox.
<img width="933" alt="image" src="https://user-images.githubusercontent.com/490574/155537314-8f5c4be8-b8ba-4015-b3ab-48ae34f191d8.png">